arch/xtensa/src/esp32/esp32_spiflash.c

```diff
@@ -131,7 +155,7 @@ static struct esp32_spiflash_s s_esp32_spiflash1 =
             .bwrite = esp32_bwrite,
             .read   = esp32_read,
             .ioctl  = esp32_ioctl,
-#if defined(CONFIG_MTD_BYTE_WRITE)
+#ifdef CONFIG_MTD_BYTE_WRITE
```

Comment:
@Ouss4 - Curiosity question: Why would the the `#ifdef` be preferred over the `#if defined`  ?

Reply:
@davids5 That was originally `#if defined(CONFIG_MTD_BYTE_WRITE) && defined(ANOTHER_CFG)` that other config was just a copy/paste typo when porting, that doesn't apply to this file.
I find "#ifdef" a little explicit compared to `#defined(ONLY_ONE_CFG)` but the two are equivalent in this case.